softwerkskammer/lib/groups/group.js

```diff
@@ -64,6 +65,10 @@ class Group {
     }
   }
 
+  canTheOrganizersBeContacted() {
+    return this.contactTheOrganizers || false;
```

Comment:
Wie wäre es hier mit `!!this.contactTheOrganizers` anstelle des Java-like `||`?

Reply:
done :)